### DISCUSSION TITLE
Show total time in task instead along the costs

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [schmidfelix/timey#4] [Ideas] [answered]
It would be helpful if the total time used by a task is available right at the task on the dashboard

![CleanShot 2021-03-23 at 12 24 37](https://user-images.githubusercontent.com/6506510/112139635-4e1fcf80-8bd3-11eb-8611-785e769a65fb.png)

**Comment by schmidfelix** [comment, marked as answer]
Great idea! In the next release there will be an option to select to show the tracked time, its costs or both 👍

**Reply by SebastiaanKloos** [reply]
Awesome!😎
